storage/src/handler.ts

```diff
@@ -0,0 +1,60 @@
+import express from 'express';
+import storage from './storage';
+
+const router: express.Router = express.Router();
+
+function logError(...args: any[]) {
+  // eslint-disable-next-line no-console
+  console.error(...args);
+}
+
+router.post('/storage/upload/sign', express.json(), async (req, res) => {
+  const { contentType } = req.body;
+  if (typeof contentType !== 'string') {
+    res.statusCode = 400;
+    res.json({ error: 'Expected contentType to be passed in body' });
+    return;
+  }
+
+  try {
+    res.json(await storage.createUpload({ contentType }));
+  } catch (err) {
+    logError(err);
+    res.sendStatus(500);
+  }
+});
+
+if (process.env.NODE_ENV !== 'production') {
```

Comment:
ok. please document this.
also please extract the path to a const so it will be clear it's being reused elsewhere. it might not work if baseUrl != /storage for example.
also while /storage/upload/sign cannot conflict with /storage/upload/:id it is only since uuidv4 can't return the string sign. might want to change this too

Reply:
`sign` and `:id` have different HTTP methods.
`baseUrl` is hardcoded here since there's no user exposed way of modifying it.
I'll document in a future PR.